shimmingtoolbox/shim/sequencer.py

```diff
@@ -93,12 +93,35 @@ def shim_sequencer(nii_fieldmap, nii_anat, nii_mask_anat, slices, coils: ListCoi
 
     # Make sure anat has the appropriate dimensions
     anat = nii_anat.get_fdata()
-    if anat.ndim != 3:
-        raise ValueError("Anatomical image must be in 3d")
+    if anat.ndim == 3:
+        pass
+    elif anat.ndim == 4:
+        logger.debug("Target anatomical is 4d, taking the average and converting to 3d")
```

Comment:
```suggestion
        logger.info("Target anatomical is 4d, taking the average and converting to 3d")
```

looks like useful info for the user

Reply:
Fixed here: [6a1a4f0](https://github.com/shimming-toolbox/shimming-toolbox/pull/389/commits/6a1a4f01047ff06dc303dc281897321e7407c0f4)